simulation_step_time: 0.01

joint_state_publish_rate: 250

robots:
- 'nj_220_2_7'

nj_220_2_7:
  start_position: [0,0,0]
  start_orientation: [0,0,0]
  foulder_path: '/home/gauss/projects/personal_ws/src/nj_220_2_7/nj_220_2_7_description/urdf'
  file_name: 'nj_220_2_7_pybullet.urdf'
  controlled_joint_name: ['joint_1', 'joint_2', 'joint_3', 'joint_4', 'joint_5', 'joint_6']
  constraints:
    - parent_body: 'nj_220_2_7'
      parent_link: 'link_2_3'
      child_body: 'nj_220_2_7'
      child_link: 'link_2_4'
      type: 'fixed'                   # joint type: prismatic, fixed, point2point, gear
      axis: [0,0,0]                   # joint axis, in child link frame
      parent_frame_position: [0,0,0]  # position of the joint frame relative to parent center of mass frame.
      child_frame_position: [0,0,0]   # position of the joint frame relative to a given child center of mass frame (or world origin if no child specified)
